src/borg/remote.py

```diff
@@ -883,6 +906,14 @@ def close(self):
             self.p.wait()
             self.p = None
 
+    def async_wait(self):
+        for resp in self.call_many('async_wait', calls=[], wait=True):
+            return resp
+
+    def async_response(self):
+        for resp in self.call_many('async_responses', calls=[], wait=True):
+            return resp
+
```

Comment:
yes, I considered that, but did not want to move too much stuff around just for that.